addon/components/o-s-s/content-panel.hbs

```diff
@@ -0,0 +1,3 @@
+<div class="oss-content-panel fx-row fx-1" ...attributes>
```

Comment:
I'm afraid we'll have issues if you specify fx-row here. I would remove `fx-row fx-1` altogether and make it the responsibility of the caller to define it. Plus you already put the `...attributes` :)

Reply:
Or we define it in a sub-div like you did in the .stories file.
let's see what the others prefer (@phndiaye  @JulienVannier66 )